testing_and_setup/compass/ocean/hurricane/spinup_time_varying_forcing.py

```diff
@@ -0,0 +1,69 @@
+import netCDF4
```

Comment:
Please put a header block on each piece of code you've written here @sbrus89 that designates a date and your name.

Reply:
Good idea, I took care of this.